module/bamqc_qualimap.nf

```diff
@@ -13,13 +13,10 @@ process run_bamqc_Qualimap {
         mode: "copy",
         enabled: true
 
-    publishDir path: "${params.workflow_log_output_dir}",
-        pattern: ".command.*",
-        mode: "copy",
-        saveAs: { "${task.process.replace(':', '/')}-${id}/log${file(it).getName()}" }
+    ext log_dir_suffix: { "-${sm_id}" }
 
     input:
-        tuple val(orig_id), val(id), path(path), val(read_length), val(sample_type)
+        tuple path(path), val(unused), val(sm_id), val(unused), val(unused), val(unused), val(unused), val(unused)
```

Comment:
I initially did this because for `bamqc_qualimap.nf` and `collectWgsMetrics.nf` it looks like we have sample, library and readgroup IDs for each input which is confusing.  You have to look back to the channel definitions in `main.nf` to see that library and readgroup are `null`.  I also found it useful during development to be able to glance and see what was being used.

I'm open to putting them all back to their original variable names, or maybe just using `unused` for library and readgroup in `bamqc_qualimap.nf` and `collectWgsMetrics.nf` and not the others?